### PR TITLE
Split splunk-forwarder-images out of splunk-forwarder-operator

### PR DESCRIPTION
**Pull request by 2uasimojo**
The shiny new https://github.com/openshift/splunk-forwarder-images repository is being split out of https://github.com/openshift/splunk-forwarder-operator. This PR configures prow for the former, moving over the relevant CI jobs.

[OSD-7270](https://issues.redhat.com/browse/OSD-7270)

### CODE REVIEW

**Comment by 2uasimojo** [comment]
/hold

Expect this to barf until https://github.com/openshift/splunk-forwarder-images/pull/1 merges.

**Comment by 2uasimojo** [comment]
> Expect this to barf until [openshift/splunk-forwarder-images#1](https://github.com/openshift/splunk-forwarder-images/pull/1) merges.

It has merged.

/retest

File: ci-operator/config/openshift/splunk-forwarder-images/openshift-splunk-forwarder-images-master.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
```
Comment (2uasimojo) [inline ci-operator/config/openshift/splunk-forwarder-images/openshift-splunk-forwarder-images-master.yaml:10]: Note to reviewers:

Yes, this is weird.

We [need](https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image) a `root` image that contains a `git` command, which apparently `ubi-minimal` does not. That image is only used to build `src`, which is what clones in the source repo, which is how we obtain the Dockerfiles etc. At that point the Dockerfiles are run `FROM base` -- i.e. the `ubi-minimal` above.

So TL;DR: The golang image doesn't figure in the splunk-(heavy)forwarder images at all. We just need to pull it (or something like it) in to make the pipeline work.

See [this slack thread](https://coreos.slack.com/archives/CBN38N3MW/p1621018598439200) for details.

**Comment by 2uasimojo** [comment]
/hold cancel

Phew.

**Review by petr-muller — APPROVED** [review]

File: ci-operator/config/openshift/splunk-forwarder-operator/openshift-splunk-forwarder-operator-master.yaml [hunk 2]
```diff
@@ -53,42 +42,6 @@ tests:
   commands: make validate
   container:
     from: src
-- as: vuln-check-operator
```
Comment (2uasimojo) [inline ci-operator/config/openshift/splunk-forwarder-operator/openshift-splunk-forwarder-operator-master.yaml:56]: I should not have removed this. Fix forthcoming.

Comment (2uasimojo) [reply ci-operator/config/openshift/splunk-forwarder-operator/openshift-splunk-forwarder-operator-master.yaml:56]: #18634
